src/include/common.h

```diff
@@ -173,6 +173,7 @@ enum eServoOutputMode : uint8_t
     som160Hz,
     som333Hz,
     som400Hz,
+    somDShot,   // DShot300
```

Comment:
This will need to go at the end of the array so it doesnt mess with the eeprom config when updating.

https://github.com/ExpressLRS/ExpressLRS/blob/567bc8c4fbb574330af70f9162586a43e2bbaaca/src/lib/CONFIG/config.h#L185

Reply:
Hopefully resolved with https://github.com/ExpressLRS/ExpressLRS/pull/2346/commits/848b198516a1b77f794a5cd71ebbe7713ccbd43f